guides/m1x/magefordev/mage-for-dev-5.html

```diff
@@ -184,7 +184,7 @@ <h3>The Global Config and Creating The Model</h3>
 &lt;/global>
 </pre>
 
-<p>The outer &lt;weblog /&gt; tag is your Group Name, which should match your module name.  &lt;class /&gt; is the BASE name all Models in the weblog group will have, also calles <strong>Class Prefix</strong>. The &lt;resourceModel /&gt; tag indicates which Resource Model that weblog group Models should use.  There's more on this below, but for now be content to know it's your Group Name, followed by a the literal string "resource".</p>
+<p>The outer &lt;weblog /&gt; tag is your Group Name, which should match your module name.  &lt;class /&gt; is the BASE name all Models in the weblog group will have, also calles <strong>Class Prefix</strong>. The &lt;resourceModel /&gt; tag indicates which Resource Model that weblog group Models should use.  There's more on this below, but for now be content to know it's your Group Name, followed by the literal string "resource".</p>
```

Comment:
Revise "calles" to "called".

Revise last sentence here as shown (or similar): "We talk more about this later on this page. For now, remember your Group Name and the literal string "resource".

Reply:
@shrielenee I've updated mage-for-dev-5.html, please check now.